scripts/av-hangouts-notifications.coffee

```diff
@@ -82,14 +113,20 @@ module.exports = (robot) ->
     # host_avatar = https://www.gravatar.com/avatar/fsd87fgds87f4387
 
     user = name: req.body.host_name, avatar: req.body.host_avatar
-    send_message CHANNELS.general, "#{req.body.title}: #{req.body.link}", user
 
     if req.body.type == "Scrum"
-      send_message CHANNELS.standup_notifications, "@channel #{req.body.title}: #{req.body.link}", user
+      send_slack_message CHANNELS.general, "#{req.body.title}: #{req.body.link}", user
+      send_slack_message CHANNELS.standup_notifications, "@channel #{req.body.title}: #{req.body.link}", user
     else if req.body.type == "PairProgramming"
       room = find_project_for_hangout(req.body.title.toLowerCase())
-      send_message CHANNELS.pairing_notifications, "@channel #{req.body.title}: #{req.body.link}", user
-      send_message room, "#{req.body.title}: #{req.body.link}", user
+
+      if room == CHANNELS.sass or room == CHANNELS.cs169
```

Comment:
In Coffeescript, `or` gets replaced by `||` during compilation.

And yes, it the title has "saas" or "cs169" in the title, it will be posted to gitter.